src/content/docs/setup/configuration/multistore-setup.mdx

```diff
@@ -9,46 +9,93 @@ import Task from '@components/Task.astro';
+- **Multiple Brands or Regions**: Operating several brands or regional storefronts, each with their own configuration, but sharing core code and content.
 
-The Acme brand is expanding its international presence and wants to target the Canadian market with two new locales:
-- **Canadian English** (en-ca)
-- **Canadian French** (fr-ca)
+There are two main approaches to multistore:
+
+1. **Multiple Domains**: Each domain points to a different commerce backend or configuration, but shares the same code and content.
+2. **Subfolders (Root Folders)**: A single domain with subfolders (e.g., `/en/`, `/fr/`) for each locale or store. Each subfolder can have its own localized content and commerce configuration.
+
+**Key features:**
+- **Editorial and Commerce Localization**: Editorial content (pages, blocks) and commerce data (products, prices, currencies) are both localized.
+- **Automated Translation Tools**: Tools are provided to automate translation of content and manage updates across languages.
+- **Configurable Headers**: Storefronts can send different headers to the commerce backend to select the correct store, language, and currency.
+- **Store Switcher**: Out-of-the-box UI enables users to switch between stores and locales.
+- **Automatic Link Management**: Links are automatically updated to point to the correct localized version as users navigate.
+
+---
+
+## How it works
+
+Adobe Commerce uses a [hierarchical structure](https://experienceleague.adobe.com/en/docs/commerce-admin/start/setup/websites-stores-views) to manage multiple stores within a single instance. This structure consists of three levels: websites, stores, and store views. Localization is managed at the store view level, allowing merchants to present the store in different languages and apply the proper currency.
+
+Each store or locale is defined in configuration files, specifying which headers to send to the commerce backend, which content folders to use, and how to structure localized content. The content hierarchy follows [Edge Delivery Services](https://experienceleague.adobe.com/en/docs/experience-manager-cloud-service/content/edge-delivery/overview) best practices, emphasizing a single-tier structure.
+
+### Content organization
+
+Content is organized primarily by language. US English files should be placed in the `en` directory of the project root. All other language- and region-specific files should be placed in a directory that is named in the format `*region-code_language-code*`. For example, the `/en/` directory contains data for the US market, while the `/en_ca/` directory contains data for Canada. French content should be placed in directories with names like `fr_fr` for France and `fr_ca` for Canada.
+
+<FileTree>
+- **/en/** _-- English store (Default root language)_
+  - placeholders/** _-- Stores JSON files for text and UI components for the English US store._
```

Comment:
Why the unpaired `**`? Same for the subsequent placeholders on line 53

Reply:
That's a mistake.